src/rp2_common/hardware_pwm/include/hardware/pwm.h

```diff
@@ -134,6 +134,21 @@ static inline void pwm_config_set_clkdiv_int(pwm_config *c, uint div) {
     c->div = div << PWM_CH0_DIV_INT_LSB;
 }
 
+/** \brief Set PWM clock divider in a PWM configuration using an 8:4 fractional value
+ *  \ingroup hardware_pwm
+ *
+ * \param c PWM configuration struct to modify
+ * \param integer 8 bit integer part of the clock divider. Must be greater than or equal to 1.
```

Comment:
It's a `uint8_t`, so the `8 bit` at the front might be superfluous?

Reply:
That text was copied mostly verbatim from the existing `int_frac` function. https://github.com/raspberrypi/pico-sdk/blob/d54104a1e5cff04979fe442cadd3018ac583d5d2/src/rp2_common/hardware_pwm/include/hardware/pwm.h#L368

Happy to change it if you'd prefer.

Reply:
Ahh, I didn't realise that sorry. Yeah, it probably makes sense to stick to what the existing doxygen comments elsewhere say, so you just need to fix "4 bit the fractional part" to say "4 bit fractional part"

Reply:
Ah, good spot.